backend/src/main/java/ro/utcn/sd/a3/command/Command.java

```diff
@@ -0,0 +1,34 @@
+package ro.utcn.sd.a3.command;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+import ro.utcn.sd.a3.dto.QuestionDTO;
+import ro.utcn.sd.a3.entity.Question;
+import ro.utcn.sd.a3.repository.QuestionRepository;
+import ro.utcn.sd.a3.service.QuestionService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+@Component
+@AllArgsConstructor
+public class Command {
+    private final QuestionService questionService;
+
+    public List<QuestionDTO> handleCommands(String command, Object o){
```

Comment:
This is not really a 100% correct implementation of the command pattern. The command pattern requires that you have an interface for your commands and several underlying implementations (which then can leverage polymorphism).